### PR TITLE
Update authors

### PR DESCRIPTION
**Pull request by sipa**

### CODE REVIEW

**Comment by sipa** [comment]
Ping @ajtowns, @real-or-random, @nickler.

**Comment by real-or-random** [comment]
ACK

**Comment by ajtowns** [comment]
Fine by me

File: bip-schnorr.mediawiki [hunk 1]
```diff
@@ -2,6 +2,8 @@
   BIP: ?
   Title: Schnorr Signatures for secp256k1
   Author: Pieter Wuille <pieter.wuille@gmail.com>
+          Jonas Nick <Jonas Nick <jonasd.nick@gmail.com>
```
Comment (jonasnick) [inline bip-schnorr.mediawiki]: there's one extra `<Jonas Nick`

Comment (sipa) [reply bip-schnorr.mediawiki]: Oops, missed this.

**Pull request merged**